NYPLCardCreator/BusinessLogic/PatronCreationInfo.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+
+// This class looks similar to UserInfo but
+// this is designed to store info ready to be send to server,
+// while UserInfo is designed to store info entered by user.
+final class PatronCreationInfo {
```

Comment:
could this be a `struct`? it's only data, so it seems to fit a struct pretty naturally.

Reply:
Done